blog post: Micro benevolences (#79)
openwebdocs/owd-website: pages/content/posts/micro-benevolences.md

The OWD team has been working to remove potential microaggressions on MDN content. For example, when we encounter words that minimize the complexity of tasks, we remove them.

## Adding micro-benevelonces

A micro-benevolence is a small act of kindness that benefits another person, without expecting reciprocity. Micro benevolences can be as simple as including culturally sensitive names in code examples or putting primary parent and secondary parent on a form instead of defaulting to cis-gendered, heteronormative, patriarchal norms.

In addition to removing potential microaggressions on MDN, the OWD team has been working to create micro-benevolences when updating code examples and adding imagery. When we update code examples to more modern code standards we also update the example itself to reflect the diversity of our readers. By increasing the diversity of cultures, genders, ages, and topics portrayed in code examples, more of our readers will feel seen and included, and more of our readers will be exposed to cultures, genders, ages, and topics of interest not their own.

### Diversifying code examples

> For example, when learning to code, you will often come across John Doe and Mike Smith in code examples, with the occasional Jane and Kimberly.  The OWD team went thru the content directory and updated these examples to better reflect the diversity of developers. The first names in a [table of names](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child) include common first names of different cultures, such as Diego, Shilpa, Caterina, Jayla, Aisha, Kyouko, and Shireen, and non-gendered names, such as Dominique, Jayla, and Gila; not just John Doe and Mike Smith, and the occasional Jane and Kimberly.  Opportunities for benevolences abound, such as a [table was a list of names and countries](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child#using_of_selector_to_fix_striped_tables) where we included country-appropriate names, and even indigenous names, like Tlayolotl for Mexico.  With a little bit of effort, and mostly awareness, we hope to make all people feel included and seen.

Reviewer: A nit + a duplicate from above
```suggestion
For example, when learning to code, you often encounter John Doe and Mike Smith in code examples, with the occasional Jane and Kimberly.  The OWD team went thru the content directory and updated these examples to better reflect the diversity of developers. The first names in a [table of names](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child) include common first names of different cultures, such as Diego, Shilpa, Caterina, Jayla, Aisha, Kyouko, and Shireen, and non-gendered names, such as Dominique, Jayla, and Gila;.  Opportunities for benevolences abound, such as a [table was a list of names and countries](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child#using_of_selector_to_fix_striped_tables) where we included country-appropriate names, and even indigenous names, like Tlayolotl for Mexico.  With a little bit of effort, and mostly awareness, we hope to make all people feel included and seen.
```
